tests/Silex/Tests/Application/FormTraitTest.php

```diff
@@ -26,6 +26,14 @@ public function testForm()
         $this->assertInstanceOf('Symfony\Component\Form\FormBuilder', $this->createApplication()->form());
     }
 
+    public function testNamedForm()
+    {
+        $builder = $this->createApplication()->namedForm('foo');
+
+        $this->assertInstanceOf('Symfony\Component\Form\FormBuilder', $builder);
+        $this->assertEquals('foo', $builder->getName());
```

Comment:
:+1: for adding :)
two minors;
`'Symfony\Component\Form\FormBuilder'` can be `FormBuilder::class`
`assertEquals` can be `assertSame`